from slither_core.exceptions import SlitherException


class SlitherCoreError(SlitherException):
    pass
